Project-3/README.md

```diff
@@ -11,67 +11,238 @@ For any question involving the use of Protege, please be sure to import:
+  Thus, although the “has characteristic” object property has the inverse functional characteristic listed likely for pragmatic purposes, I argue that it would be       better to dispense with it if one’s aim is to represent relations that are many to one, one to many, or many to many. 
+
 ```
 
 3. Model the following natural language expressions using terms from BFO and RO; you are welcome to introduce new terms where needed:  
 ```
   (a) Sally has an arm Tuesday but does not have an arm Wednesday. 
+  Sally is instance_of object
+  Tuesday is instance_of one-dimensional temporal region 
+  Wednesday is an instance of one-dimensional temporal region 
+  Arm is an instance of fiat object part
+  “Sally participates in having at least one arm on Tuesday” is an instance of occurrent 
+  “Sally participates in having no arms on Wednesday” is an instance of occurrent
+  “Sally participates in having at most one arm on Wednesday” is an instance of occurrent
+  Due to the vagueness of the statement above, we have narrowed the possible answers to the following two cases:
+  “Sally participates in having at least one arm on Tuesday precedes Sally participates in having no arms on Wednesday”
+  “Sally participates in having at least one arm on Tuesday precedes Sally participates in having at most one arm on Wednesday”
+
   (b) Every liver has some cell as part at all times it exists.
-  (c) John was a child, then an adult, then a senior. 
+  Search liver here: https://ontobee.org
+  We already have cell in the ontology.
+  Liver is a class in Uberon (UBERON_0002107), which is a subclass of material entity. 
+  https://ontobee.org/ontology/UBERON?iri=http://purl.obolibrary.org/obo/UBERON_0002107
+  Argument according to authority BS for the object property has_part_at_all_times: https://youtu.be/fkkWkTIxrNQ
+  Liver has_part_at_all_times Cell
+
+  (c) John was a child, then an adult, then a senior.
+  John is an instance_of object
+  childhood is an instance_of occurrent.
+  adulthood is an instance_of occurrent.
+  seniorhood is an instance_of occurrent.
+  “John participates in childhood precedes John participates in adulthood which precedes John participates in seniorhood.”
```

Comment:
It seems better if we let "John's childhood", "John's adulthood" and "John's seniorhood" as instances of occurrent. After all, "childhood", "adulthood" and "seniorhood" look more like classes instead of instances.